runtime/global.go

```diff
@@ -43,13 +44,24 @@ func Prelude() values.Scope {
 
 // Eval accepts a Flux script and evaluates it to produce a set of side effects (as a slice of values) and a scope.
 func Eval(ctx context.Context, flux string, opts ...flux.ScopeMutator) ([]interpreter.SideEffect, values.Scope, error) {
-	h := parser.ParseToHandle([]byte(flux))
-	return Default.evalHandle(ctx, h, opts...)
+	h, err := parser.ParseToHandle([]byte(flux))
+	if err != nil {
+		return nil, nil, err
+	}
+	return Default.Eval(ctx, h, opts...)
 }
 
 // EvalAST accepts a Flux AST and evaluates it to produce a set of side effects (as a slice of values) and a scope.
 func EvalAST(ctx context.Context, astPkg *ast.Package, opts ...flux.ScopeMutator) ([]interpreter.SideEffect, values.Scope, error) {
```

Comment:
Who calls this method? Should callers of this method just use the runtime interface instead? 

What is the reason for not changing this interface to expect an ASTHandle vs the ast.Package?

Reply:
Tasks need to use this method to evaluate an AST, it's related to processing options. They don't need to actually submit a request, but they do need to manipulate and evaluate a Go AST, and look at the resulting scope produced by evaluation.

Would it make sense to rename this `EvalGoAST`?

Reply:
I like the name `EvalAST` as the function signature makes it clear it must be the Go types. Knowing why we still need it is helpful.